tox.ini

```diff
@@ -811,59 +809,38 @@ setenv =
-        k3d cluster start syft'
-
-    sleep 10
-    bash -c "kubectl --context k3d-syft create namespace syft || true"
-
-    # if syft version is local, then install local helm charts
-    # else install the helm charts from the openmined gh-pages branch
     bash -c 'if [[ $SYFT_VERSION == "local" ]]; then \
         echo "Installing local helm charts"; \
-        bash -c "cd packages/grid/helm && helm install --kube-context k3d-syft --namespace syft syft ./syft --set global.useDefaultSecrets=true"; \
+        helm install ${CLUSTER_NAME} ./helm/syft -f ./helm/values.dev.yaml --kube-context k3d-${CLUSTER_NAME} --namespace syft --create-namespace; \
     else \
         echo "Installing helm charts from repo for syft version: ${SYFT_VERSION}"; \
-        bash -c "helm repo add openmined https://openmined.github.io/PySyft/helm && helm repo update openmined"; \
-        bash -c "helm install --kube-context k3d-syft --namespace syft syft openmined/syft --version=${SYFT_VERSION} --set global.useDefaultSecrets=true"; \
+        helm repo add openmined https://openmined.github.io/PySyft/helm; \
+        helm repo update openmined; \
+        helm install ${CLUSTER_NAME} openmined/syft --version=${SYFT_VERSION} -f ./helm/values.dev.yaml --kube-context k3d-${CLUSTER_NAME} --namespace syft --create-namespace; \
     fi'
 
     ; wait for everything else to be loaded
-    bash packages/grid/scripts/wait_for.sh service frontend --context k3d-syft --namespace syft
-    bash -c '(kubectl logs service/frontend --context k3d-syft --namespace syft -f &) | grep -q -E "Network:\s+https?://[a-zA-Z0-9.-]+:[0-9]+/" || true'
-    bash packages/grid/scripts/wait_for.sh service mongo --context k3d-syft --namespace syft
-    bash packages/grid/scripts/wait_for.sh service backend --context k3d-syft --namespace syft
-    bash packages/grid/scripts/wait_for.sh service proxy --context k3d-syft --namespace syft
-    bash -c '(kubectl logs service/backend --context k3d-syft --namespace syft -f &) | grep -q "Application startup complete" || true'
-
+    bash -c './scripts/wait_for.sh service frontend --context k3d-$CLUSTER_NAME --namespace syft'
+    bash -c '(kubectl logs service/frontend --context k3d-$CLUSTER_NAME --namespace syft -f &) | grep -q -E "Network:\s+https?://[a-zA-Z0-9.-]+:[0-9]+/" || true'
+    bash -c './scripts/wait_for.sh service mongo --context k3d-$CLUSTER_NAME --namespace syft'
+    bash -c './scripts/wait_for.sh service backend --context k3d-$CLUSTER_NAME --namespace syft'
+    bash -c './scripts/wait_for.sh service proxy --context k3d-$CLUSTER_NAME --namespace syft'
+    bash -c '(kubectl logs service/backend --context k3d-$CLUSTER_NAME --namespace syft -f &) | grep -q "Application startup complete" || true'
 
     # Run Notebook tests
     tox -e e2e.test.notebook
 
-    # Cleanup
-    bash -c "k3d cluster delete syft || true"
-    bash -c "docker volume rm k3d-syft-images --force || true"
+    bash -c "k3d cluster delete ${CLUSTER_NAME} || true"
```

Comment:
MAYBE: we could use tox -e dev.k8s.destroy itself here?

Reply:
Destroy calls cleanup. Cleanup = remove images, deployment and namespace. Destroy = destroy the whole k3d cluster.